identity_credential/src/validator/vp_jwt_validation/jwt_presentation_validator.rs

```diff
@@ -62,39 +56,36 @@ where
   /// - the JWT can be decoded into semantically valid presentation.
   /// - the expiration and issuance date contained in the JWT claims.
   /// - the holder's signature.
-  /// - the relationship between the holder and the credential subjects.
-  /// - the signatures and some properties of the constituent credentials (see [`CredentialValidator`]).
   ///
   /// Validation is done with respect to the properties set in `options`.
   ///
   /// # Warning
-  /// The lack of an error returned from this method is in of itself not enough to conclude that the presentation can be
-  /// trusted. This section contains more information on additional checks that should be carried out before and after
-  /// calling this method.
+  /// * This method does NOT validate the constituent credentials, nor the relationship between the
+  /// credentials' issuers and the presentation holder.
```

Comment:
```suggestion
  /// * This method does NOT validate the constituent credentials.
```

Reply:
It doesn't hurt to leave that extra bit of information in, actually. Why would you like to remove it?

Reply:
I thought since we have it now in the options of the credential validator, it's not part of the presentation validator. But it's fine to keep it.